src/main/resources/ESAPI.properties

```diff
@@ -0,0 +1,431 @@
+#
+# Copyright 2005-2016 The Kuali Foundation
+#
+# Licensed under the Educational Community License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.opensource.org/licenses/ecl2.php
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+# CU Customization: Added the Rice 2.3.13 version of this file.
```

Comment:
Looks like this was copied from the cynergy-standalone version of the file, where I had added this comment to clarify that it was an addition or overlay from existing Rice code. Should this be updated or removed accordingly on the KFS side?